methods/matching/find_pairs.py

```diff
@@ -38,7 +40,7 @@ def find_match_iteration(
     # Methodology 6.5.7: For a 10% sample of K
     k_set = pd.read_parquet(k_parquet_filename)
     k_subset = k_set.sample(
-        frac=0.1,
+        frac=1,
```

Comment:
Isn't this just the same as k_subset = k_set?

Reply:
Yes; I didn't clean this up yet as I wasn't sure if we definitely wanted to change to 100% of K instead of 10%.